src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,79 +24,94 @@ redirects:
   - /docs/alerts-applied-intelligence
 ---
 
-import accountsIssueFeedforOverviewPage from 'images/accounts_screenshot-full_issue-feed-for-overview-page.png'
+import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong... right when they're starting to go wrong.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relics alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
 
-Want to try Alerts & AI? Join us! It's free... forever!
+Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that alerts your team to potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
 
-<ButtonGroup>
-  <ButtonLink
-    role="button"
-    to="https://newrelic.com/signup"
-    variant="primary"
-  >
-    Sign up for free
-  </ButtonLink>
-</ButtonGroup>
+Alerts and Applied intelligence were created for busy DevOps teams: it reduces noise and makes sure you never miss an issue. 
 
-Already have an account? [Login](https://one.newrelic.com).
+## How it all works 
```

Comment:
we should avoid having an h2 followed directly by an h3. we should try to give a little context of what will be in that h2 section before going into the h3s

Reply:
done!